content/docs/_index.md

```diff
@@ -9,7 +9,7 @@ icon: docs
 
 In addition, you might find these links interesting:
 
-- Check out our [Help](/help) section, which includes our [Operations Guide](/help/ops), our [FAQ](/help/faq), and our [glossary](/help/glossary).
+- Check out our [Help](/help) section, which includes our [Operations Guide](/help/ops), our [FAQ](/help/faq), and our [glossary](/docs/reference/glossary).
```

Comment:
Should remove this paragraph as we will no longer have a Help section.

Reply:
This is addressed in #4099 I didn't do it here to avoid merge conflicts. The PR making the FAQ a top level section will make the final changes to this file.